shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformViewsChannel.java

```diff
@@ -0,0 +1,479 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.embedding.engine.systemchannels;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.Map;
+
+import io.flutter.embedding.engine.dart.DartExecutor;
+import io.flutter.plugin.common.MethodCall;
+import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.StandardMethodCodec;
+
+/**
+ * System channel that sends 2-way communication between Flutter and Android to
+ * display Android Views within Flutter.
```

Comment:
This channel and the PlatformViewsController isn't displaying the views, it configures them, and passes input events.

Maybe replace "to display Android Views within Flutter" with "to facilitate embedding of Android Views within a Flutter application".

Reply:
done